src/Psl/PseudoRandom/float.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\PseudoRandom;
+
+use Psl;
+use Psl\Math;
+
+/**
+ * Returns a cryptographically secure random float in the range from 0.0 to 1.0.
+ *
+ * @throws Psl\Exception\RuntimeException If it was not possible to gather sufficient entropy.
```

Comment:
`PseudoRandom\float`, and `PseudoRandom\int` don't return cryptographically secure randoms, and the exception is never thrown.